Kiosk/App/AppDelegate+GlobalActions.swift

```diff
@@ -28,23 +31,37 @@ extension AppDelegate {
         helpButton.setTitle("Help", for: .normal)
         helpButton.rx.action = helpButtonCommand()
         window?.addSubview(helpButton)
-        helpButton.alignTop(nil, leading: nil, bottom: "-24", trailing: "-24", to: window)
+        helpButton.alignTop(
+            nil,
+            leading: nil,
+            bottom: "-24",
+            trailing: "-24",
+            to: window
+        )
```

Comment:
What is the max line length you're using for this? We use 120 everywhere - ( yes prettier people, we walk up-hill in all directions ;) )

Reply:
Personally, I'd go with defaulting 120 simply because 4 spaces is convention for Swift indentation, as you have a typed language (with a very verbose legacy) with a lot of whitespace